src/sentry/static/sentry/app/routes.jsx

```diff
@@ -370,6 +375,7 @@ function routes() {
             component={errorHandler(OrganizationContext)}
           >
             <Route component={errorHandler(OrganizationSettingsLayout)}>
+              {hooksOrgSettingsRoutes}
```

Comment:
Just a note we will want a hook in `routes:organization` when we release orgMembersView to GA

Reply:
You mean when the new view goes out to GA (https://github.com/getsentry/sentry/pull/6308), it'll still be served from `/organizations/orgId/members`?

Reply:
yeah